webview-ui/src/components/ResolutionsPage/ModifiedFile/EnhancedDiffRenderer.tsx

```
@@ -0,0 +1,309 @@
+    (line: DiffLine, index: number) => {
+      const actualIndex = shouldVirtualize ? visibleRange.start + index : index;
+      const highlightedContent = isScrolling ? line.content : getHighlightedContent(line);
+
+      return (
+        <div
+          key={actualIndex}
+          className={`diff-line ${line.type}`}
+          style={shouldVirtualize ? { height: lineHeight } : undefined}
+        >
+          <span className="diff-line-number">{line.lineNumber}</span>
+          <span className="diff-content" dangerouslySetInnerHTML={{ __html: highlightedContent }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)